src/App.tsx

```diff
@@ -1,109 +1,12 @@
-import React from 'react';
-
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
```

Comment:
it's better to prepare data in app component and pass it as props where it's needed